<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SoftwareVersionDialog</class>
 <widget class="QDialog" name="SoftwareVersionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>217</width>
    <height>118</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget_softwareVersion">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>0</y>
     <width>130</width>
     <height>111</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QPushButton" name="pushButton_addNewVersion">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>75</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить
новую</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_removeSelectedVersion">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>75</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить
выбранную</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
